jenkins/deployment/k8s_deployment/cortx-cluster.groovy

```diff
@@ -0,0 +1,143 @@
+pipeline {
+    agent {
+        node {
+            label 'docker-centos-7.9.2009-node'
+        }
+    }
+    
+    options {
+        timeout(time: 600, unit: 'MINUTES')
+        timestamps()
+        buildDiscarder(logRotator(daysToKeepStr: '20', numToKeepStr: '20'))
+        disableConcurrentBuilds()
+    }
+    environment {
+        CORTX_RE_BRANCH = "kubernetes"
+        CORTX_RE_REPO = "https://github.com/Seagate/cortx-re/"
+        DOCKER_IMAGE_LOCATION = "https://github.com/Seagate/cortx-re/pkgs/container/cortx-all"
+    }
+    parameters {
+        string(name: 'CORTX_RE_BRANCH', defaultValue: 'kubernetes', description: 'Branch or GitHash for Cluster Setup scripts', trim: true)
+        string(name: 'CORTX_RE_REPO', defaultValue: 'https://github.com/Seagate/cortx-re/', description: 'Repository for Cluster Setup scripts', trim: true)
+        string(name: 'CORTX_IMAGE', defaultValue: 'cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-latest-kubernetes', description: 'CORTX-ALL image', trim: true)
+        string(name: 'SNS_CONFIG', defaultValue: '1+0+0', description: 'sns configuration for deployment. Please select value based on disks available on nodes.', trim: true)
+        string(name: 'DIX_CONFIG', defaultValue: '1+0+0', description: 'dix configuration for deployment. Please select value based on disks available on nodes.', trim: true)
+
```

Comment:
Can we add a comment here that `hosts` is a manually configured parameter for the job?

Reply:
Updated comment. Please review.